policyengine_us/parameters/gov/states/mn/tax/income/amt/income_fraction.yaml

```diff
@@ -11,3 +11,5 @@ metadata:
     href: https://www.revenue.state.mn.us/sites/default/files/2023-02/m1mt_21.pdf#page=1
```

Comment:
I think we can use this parameter instead in the variable:
policyengine_us/parameters/gov/irs/income/amt/exemption/phase_out/rate.yaml

We can delete this file